include/uapi/sound/sof/abi.h

```diff
@@ -27,7 +27,7 @@
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
 #define SOF_ABI_MINOR 2
-#define SOF_ABI_PATCH 0
```

Comment:
I didn't get why this was necessary, it's the first time we bump the patch version and I don't see any real change that would motivate this version change. What am I missing?

Reply:
If this doesn't have a userspace yet, then agreed no need to change. I thought you had argued the reserve and I was assuming there are users out there. Typically you'd bump the patch level for bugfixes that don't change the interface but that are beneficial for the interface users (i.e. application could check for sufficient patch level to ensure a certain bugfix is in the kernel that is being used).

Reply:
typo: s/reserve/reverse/ 

But yeah, looking at ALSA versioning practises used (e.g. upstream 9c7066aef4a5eb8e4063de28f06c508bf6f2963a), patch level is not updated for bugfixes, so I'd say let's follow this approach for SOF as well unless we specifically agree otherwise. So Seppo if you can change the patch back, thanks...? :)